cmd/pg-schema-diff/plan_cmd.go

```diff
@@ -591,3 +597,19 @@ func planToJsonS(plan diff.Plan) string {
 	}
 	return string(jsonData)
 }
+
+func planToSqlS(plan diff.Plan) string {
```

Comment:
Can we materialize the warnings and timeouts in the plan as well?

The warnings could be added as comments. The timeouts could be set via set statement timeout command?